star.cpp

```diff
@@ -471,7 +479,7 @@ bool Star::Find(const usImage *pImg, int searchRegion, int base_x, int base_y, F
 
 bool Star::Find(const usImage *pImg, int searchRegion, FindMode mode, double minHFD, unsigned short saturation, StarFindLogType loggingControl)
 {
-    return Find(pImg, searchRegion, X, Y, mode, minHFD, saturation, loggingControl);
+    return Find(pImg, searchRegion, X, Y, mode, minHFD, pFrame->pGuider->GetMaxStarHFD(), saturation, loggingControl);

```

Comment:
let's keep this method decoupled from Guider -- have the caller pass maxHDF in addition to minHFD.  It will be more consistent that way having them both be passed explicitly rather than one explcit and one implicit.